src/common/queries/company-gateways.ts

```diff
@@ -60,3 +74,30 @@ export function useBlankCompanyGatewayQuery() {
     { staleTime: Infinity, enabled: isAdmin }
   );
 }
+
+export const useBulk = () => {
```

Comment:
`function` style of declaration, please.

Reply:
Changed!